setup.py

```diff
@@ -21,6 +21,7 @@
     author_email='info@regulusweb.com',
     install_requires=[
         'django-honeypot==0.6.0',
+        'django-crispy-forms==1.6.1',
```

Comment:
Prefer to use `>=1.6.1` so that we don't hit dependency conflicts on newer projects.